assets/dev/js/editor/components/browser-import/container-factory.js

```diff
@@ -1,4 +1,4 @@
-import ElementModel from '../../elements/models/element';
+const ElementModel = require( 'elementor-elements/models/element' );
```

Comment:
Why not `export default ElementModel;` in element module?

Reply:
Because the module is already using CommonJS syntax `module.exports = ElementModel;`, just like most of our current code. Given that, I think it would be more appropriate to use CommonJS and not ES6 syntax for importing the module as well, to align with the existing exports syntax and to ensure consistency in the codebase.